app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -60,191 +64,80 @@ keywords:
-
-## Already a qualified teacher?
-
-If you're already a qualified teacher, find out how you can [teach in England if you completed your teacher training outside the UK](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
-
-## Why train to teach in England?
-
-Teacher training in England is practical, hands-on and supportive. As an international applicant, you’ll have the opportunity to train at a world class institution where creativity and innovation is valued, gaining lots of classroom experience along the way.
-
-When you have successfully completed your training, you will be awarded [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts), a professional qualification valued globally.
-
-QTS will allow you to apply for teaching jobs in all schools in England.    
-
-[Read teachers' stories about the rewards of training to teach in England](/blog).
-
-## Other ways to train
-
-You can also apply to [train to teach in Wales](https://educators.wales/home), [train to teach in Scotland](https://teachinscotland.scot/) or [train to teach in Northern Ireland](https://www.education-ni.gov.uk/articles/initial-teacher-education-courses-northern-ireland).
-
-If you don’t want to come to England to study you can apply to train for international qualified teacher status (iQTS). This meets the same high standards as [English qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) and, if successfully completed, leads to the automatic award of QTS.
+- you may need to extend or switch your visa to live and work in the UK long-term
 
-For teachers who qualified outside the UK, as well as those with teaching experience, [other routes to qualified teacher status (QTS)](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk) are also open.
+## 1. Check your qualifications
 
-## Check your qualifications and get help
-
-Before you apply, you must [check your qualifications](/non-uk-teachers/non-uk-qualifications). If you gained your qualifications outside the UK, we can help you check they meet the required standard in England. 
+Before you apply, you must [check your qualifications](/non-uk-teachers/non-uk-qualifications). If you gained your qualifications outside the UK, we can help you check if they meet the required standard in England. 
 
 To get help with your qualifications, you can use the live chat service at the bottom of this page. Or you can call us on +44 800 389 2500. Calls will be charged at your country's standard rate.
 
 We're available Monday to Friday between 8:30am and 5:30pm (UK local time) except on [bank holidays](https://www.gov.uk/bank-holidays) in England. 
 
 When you speak to us, we can also tell you if you're eligible for a [teacher training adviser](/teacher-training-advisers). 
```

Comment:
I think we should remove this link - we want people ringing us to check their eligibility, not going through the form on the TTA page.

If needed, we could add a sentence to describe what TTAs can help with?

Reply:
I'm a bit confused by this... presumably the TTAs will tell them if they're eligible for a TTA before giving them advice on qualifications?